### ISSUE TITLE
allow focusable content with tabindex=-1 inside hidden content

### ISSUE THREAD

**Issue by DavidMacDonald** [wet-boew/wet-boew-wpss#121]
```
<div aria-hidden="true" ...>
       <button  tabindex="-1" ...

Currently fails
WCAG_2.0-SC4.1.2

**Comment by irwink** [comment]
Tool was not checking tabindex attribute, will be fixed in next release.

**Comment by DavidMacDonald** [comment]
Another example

 SC4.1.2: Name, Role, Value | Focusable content inside aria-hidden tag

<div id="cat_select_form"> <select id="cat_select" data-bloglink="https://cepeo.on.ca/nouvelles/" data-select2-id="cat_select" tabindex="-1" class="select2-hidden-accessible" aria-hidden="true">

**Comment by irwink** [comment]
Corrected in version 6.13.0.
